### DISCUSSION TITLE
posts/reading-notes-of-building-a-second-brain/

### DISCUSSION THREAD

**Discussion by giscus** [waynerv/waynerv.github.io#13] [Announcements]
# posts/reading-notes-of-building-a-second-brain/

这本书出版于六月份，花了些时间看了大半。笔记中掺杂了一些个人的总结和看法，一般会放在原文内容归纳的后面。

https://add-giscus-support.blog-2os.pages.dev/posts/reading-notes-of-building-a-second-brain/

**Comment by waynerv** [comment]
test giscus

**Reply by waynerv** [reply]
Looks good

**Comment by wangyunzi** [comment]
hello，大佬

**Comment by scootflynn** [comment]
超棒中肯的个人总结！作者有意识的强调体系重要，工具次要，更主要是因为技术发展太快了，这会儿生成式AI的爆发，他在推广观念打底，才更高效。
